origin.spec

```diff
@@ -17,7 +17,7 @@
 # this is the version we obsolete up to. The packaging changed for Origin
 # 1.0.6 and OSE 3.1 such that 'openshift' package names were no longer used.
 %global package_refector_version 3.0.2.900
-%global golang_version 1.6.2
+%global golang_version 1.8.3
```

Comment:
1.8.3 is tagged for RHEL, but I don't think we technically need that exact version. Will change.